deployment/articles-dxp/01-saml/02-setting-up-identity-provider.markdown

```diff
@@ -0,0 +1,160 @@
+---
+header-id: setting-up-liferay-as-a-saml-identity-provider
+---
+
+# Setting up @product@ as a SAML Identity Provider
+
+An identity provider is a trusted provider that provides single sign-on for
+users to access other websites. A service provider is a website that hosts
+applications and grants access only to identified users with proper credentials.
+Liferay Portal 6.1 EE and later versions support SAML 2.0 integration via the 
+[Liferay SAML 2.0 Provider](https://web.liferay.com/marketplace/-/mp/application/15188711)
```

Comment:
Change: The name of the app is "Liferay Connector to SAML 2.0".

Reply:
Changed (locally). I'll be pushing all these commits when I'm done with the review.